### ISSUE TITLE
[BUG]: Some notebook & scripts in models don't run

### ISSUE THREAD

**Issue by tzemicheal** [nv-morpheus/Morpheus#1052]
### Version

23.07

### Which installation method(s) does this occur on?

Docker

### Describe the bug.

   - A broken reference to clx.analytics.periodicity_detection in dfp-hamma inference script. 
   - ABP notebook metric score needs to change to numpy inorder to call `sklearn.metric` methods.
   - Invalid `x_val` parameter in the `AutoEncoder` model in Hammah notebook.

### Code of Conduct

- [X] I agree to follow Morpheus' Code of Conduct
- [X] I have searched the [open bugs](https://github.com/nv-morpheus/Morpheus/issues?q=is%3Aopen+is%3Aissue+label%3Abug) and have found no duplicates for this bug report
